index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ________ *Stefaanb* Contributor card START ________  -->
+         <div class="card">
+          <p class="name">Stefaan Beernaert</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="#" target="_blank">none</a>
```

Comment:
Replace `USERNAME` with your twitter username. If you don't have twitter, replace `twitter` with `github` and put your github username instead of twitter username.
```suggestion
              <a href="https://twitter.com/USERNAME" target="_blank">@USERNAME</a>
```